tony-core/src/main/avro/TaskStarted.avsc

```diff
@@ -5,6 +5,7 @@
   "fields": [
     {"name": "taskType", "type": "string"},
     {"name": "taskIndex", "type": "int"},
-    {"name": "host", "type": "string"}
+    {"name": "host", "type": "string"},
+    {"name": "containerID", "type": ["null","string"], "default": null, "doc":  "Container  id"}
```

Comment:
I don't think containerId could be null ?

Reply:
done changes as suggested